src/micro_http/src/server.rs

```diff
@@ -379,6 +379,19 @@ impl HttpServer {
         Ok(parsed_requests)
     }
 
+    /// This function is responsible with flushing any remaining outgoing
+    /// requests on the server.
+    ///
+    /// Note that this function can block the thread on write, since the
+    /// operation is blocking.
+    pub fn flush_outgoing_writes(&mut self) {
```

Comment:
I think we should also call close() on the fds after writing all data.

Reply:
is it necessary? doesn't the same thing happen (process buffers are flushed) during/because-of `libc::exit()`?

Reply:
I agree with @acatangiu

Reply:
I agree this also works, but as a best practice we could handle cleanup ourselves.